docs/develop/python/message-passing.mdx

```diff
@@ -486,6 +484,51 @@ You can silence these warnings on a per-handler basis by passing the `unfinished
 
 See [Finishing handlers before the Workflow completes](/encyclopedia/workflow-message-passing#finishing-message-handlers) for more information.
 
+
+### Use `@workflow.init` to operate on Workflow input before any handler executes
```

Comment:
nit: "message handler" ?

Reply:
I think it's consistent as-is.

```
40:## Write message handlers {#writing-message-handlers}
53:### Query handlers {#queries}
90:### Signal handlers {#signals}
120:### Update handlers and validators {#updates}
336:## Message handler patterns {#message-handler-patterns}
347:### Use async handlers {#async-handlers}
441:#### Use wait conditions in handlers {#wait-in-message-handler}
461:#### Ensure your handlers finish before the Workflow completes {#wait-for-message-handlers}
490:### Use `@workflow.init` to operate on Workflow input before any handler executes
528:### Use `asyncio.Lock` to prevent concurrent handler execution {#control-handler-concurrency}
574:## Message handler troubleshooting {#message-handler-troubleshooting}
646:## Dynamic components {#dynamic-handler}
667:### Set a dynamic Signal, Query, or Update handler {#set-a-dynamic-signal}
```